### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by hunchback**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,98 @@
...
+- sending monitor-event is ovsdb semantics
+
+## To start developing ovsdb-etcd
+
+You must have a working [Go Environment](https://golang.org/doc/install).
+
+Then clone ovn-kubernetes from our fork:
+
+```bash
+git clone https://github.com/ibm/ovn-kubernetes
```
Comment (roytman) [inline README.md:49]: git checkout ovsdb-etcd2

File: README.md [hunk 2]
```diff
@@ -1,38 +1,98 @@
...
+- process ovsdb semantics on memory
+- etcd post the post-transaction **rows**
+- releasing the global lock
+- sending the transaction-response in ovsdb semantics
+
+An ovsdb monitor enrollment is implemented as:
+- receiving monitor-request in ovsdb semantics
+- enrolling to get etcd events per **tables**
+- sending monitor-response in ovsdb semantics
+
```
Comment (roytman) [inline README.md:36]: in order to support cross events consistency, ovsdb-etcd server monitors entire backend database and propagates all etcd events as a single OVSDB update.

File: README.md [hunk 3]
```diff
@@ -1,38 +1,98 @@
...
+```
+export CR_PASSWORD=<DOCKER_PASSWORD>
+export CR_USERNAME=<DOCKER_USENAME>
+export OVSDB_ETCD_REPOSITORY=$CR_USERNAME
+```
+Login to docker and push (optional):
+
+```
+echo $CR_PAT | docker login $CR_REPO -u $CR_USERNAME --password-stdin
```
Comment (roytman) [inline README.md:75]: $CR_PASSWORD

File: README.md [hunk 4]
```diff
@@ -1,38 +1,98 @@
...
+
+```
+export OVNDB_ETCD_TCPDUMP='false'
+```
+
+Deploy onto a kind cluster:
+
+
+```bash
+pushd ovsdb-etcd
```
Comment (roytman) [inline README.md:89]: why do we need all thes pusgd ovsdb-etcd/popd ? if user works with ovsdb-etcd so he is cd to the ovsdb-etcd and works from its root dir

File: README.md (hunk 1 above)
Comment (hunchback) [reply README.md:49]: but I have it already ... see:

```
git clone https://github.com/ibm/ovn-kubernetes
pushd ovn-kubernetes
git checkout ovsdb-etcd2
export OVN_KUBERNETES_ROOT=$PWD
popd
```

File: README.md (hunk 3 above)
Comment (hunchback) [reply README.md:75]: fixed

File: README.md (hunk 2 above)
Comment (hunchback) [reply README.md:36]: updated accordingly

File: README.md (hunk 4 above)
Comment (hunchback) [reply README.md:89]: removed

File: README.md (hunk 1 above)
Comment (roytman) [reply README.md:49]: oops

File: README.md [hunk 5]
```diff
@@ -1,38 +1,94 @@
...
+- catching the global lock
+- etcd get the pre-transaction **tables**
+- process ovsdb semantics on memory
+- etcd post the post-transaction **rows**
+- releasing the global lock
+- sending the transaction-response in ovsdb semantics
+
+An ovsdb monitor enrollment is implemented as:
+- receiving monitor-request in ovsdb semantics
+- enrolling to get etcd events per entire **database** (to support cross database consistency)
```
Comment (roytman) [inline README.md:34]: could you chang please  "cross database consistency" to "cross tables consistency"

**Pull request merged**